src/status_im/common/qr_codes/view.cljs

```diff
@@ -44,6 +44,7 @@
   [network]
   (case network
     :ethereum "eth:"
```

Comment:
I think we should get some helped methods for managing network ids for addresses, like `eth:`, `arb:` etc. 

I used strings in another PR. We can make this a constant, and then get a helper function to give a network id, something like:

```clj
(defn network-ids [network-id-constants]
  (join network-id-constants ":"))

(network-ids [constants/eth-network-id constants/arb-network-id])
```

Reply:
The constants are already present in this PR: https://github.com/status-im/status-mobile/pull/19150/files

Reply:
Thanks @shivekkhurana I will update it